src/test/resources/Hotels/HotelsCheckIn.feature

```diff
@@ -0,0 +1,24 @@
+Feature: verification of sorting of HotelsPrice
```

Comment:
You should have some tag like @regression or @hotels for Feature file level at Line 1